Samples/iOS-Swift/iOS-Swift/SentrySDKWrapper.swift

```diff
@@ -60,6 +60,7 @@ struct SentrySDKWrapper {
         options.attachViewHierarchy = enableAttachViewHierarchy
         options.enableTimeToFullDisplayTracing = enableTimeToFullDisplayTracing
         options.enablePerformanceV2 = enablePerformanceV2
+        options.enableAppHangTrackingV2 = enableAppHangTrackingV2
```

Comment:
Needs the launch arg added to the scheme. I added it in b41924378

Reply:
Thank you!